### ISSUE TITLE
Blog RSS feed is empty

### ISSUE THREAD

**Issue by dmathieu** [open-telemetry/opentelemetry.io#4440]
**What happened?**

The blog's RSS feed is empty: https://opentelemetry.io/blog/index.xml
There is a link to a feed in the blog itself: https://opentelemetry.io/blog/2024/index.xml
But that URL includes the 2024 year. There shouldn't be a need to update the feed's address every year :)

**What did you expect would happen?**

The RSS feed includes recent entries.

**Comment by svrnm** [comment]
Thanks @dmathieu 

@chalin PTAL

**Comment by chalin** [comment]
Yes, this is a bug. The following illustrates how `blog/index.xml` used to contain all entries before Docsy switched to using the Hugo RSS template:

- https://deploy-preview-3795--opentelemetry.netlify.app/blog/index.xml

Docsy issue:

- https://github.com/google/docsy/issues/1996

**Comment by chalin** [comment]
To all who have given a thumbs up to the above commet. Please add your thumbs up to the Docsy issue instead if you'd like for it to count during the Docsy-issue review process :). Thanks!

**Comment by svrnm** [comment]
@chalin would it be possible to workaround this with a redirect? Not a perfect solution since we would need to update it once per year but maybe a workaround until the upstream issue is resolved?

**Comment by chalin** [comment]
That's worth a try.

**Comment by chalin** [comment]
I'm going to close this since it was resolved by #5520. I'll be updating the feed to point to the 2025 section via:

- #5873

**Comment by chalin** [comment]
We'll integrate the "real" fix once Docsy gets updated.
